azure-pipelines/pull-request-validation/common.yml

```diff
@@ -2,6 +2,7 @@
 # Variable: 'MOCK_API_URL' was defined in the Variables tab
 # Description: Assemble and run unit test
 name: $(date:yyyyMMdd)$(rev:.r)
+trigger: none
```

Comment:
Reason: Avoids duplicate pipeline run for every push

Reply:
I guess you could've made this change in a separate PR.

Reply:
yes, sorry about that...